Gemfile.lock

```diff
@@ -0,0 +1,210 @@
+GEM
```

Comment:
@deivid-rodriguez I agree with you that it brings trouble with 3rd party dependency changes. But this is a feature and not a bug 😉 . If commit the lock would find this issues and often have PRs/commits with dependency updates. Nearly every gem don't check in there Gemfile.lock. Event `bundle gem`'s gem scaffold tells you not do that. I disagree with this change, until some more/better reasons or all other maintainers will agree with it. Please remove it and if you still want a discussion create a new separate PR.